pulsar/json_schemas/server.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://asyncapi.com/bindings/pulsar/server.json",
+  "title": "Server Schema",
+  "description": "This object contains server information of Pulsar broker. This object contains additional information not possible to represent within the core AsyncAPI specification.",
```

Comment:
Let's clarify that this `server` binding specify Pulsar `cluster`/`tenant` level config here too.

Reply:
Done